src/interfaces.ts

```diff
@@ -29,32 +37,76 @@ export interface AggregateVerifierLoginParams extends BaseLoginParams {
 export type LoginParams = SubVerifierDetailsParams | AggregateVerifierLoginParams;
 export type UserInfo = TorusVerifierResponse & LoginWindowResponse;
 
+export interface IdTokenLoginParams {
+  /**
+   * Name of the verifier created on Web3Auth Dashboard. In case of Aggregate Verifier, the name of the top level aggregate verifier.
+   */
+  verifier: string;
+
+  /**
+   * Unique Identifier for the User. The verifier identifier field set for the verifier/ sub verifier. E.g. "sub" field in your on jwt id token.
+   */
+  verifierId: string;
+
+  /**
+   * The idToken received from the Auth Provider.
+   */
+  idToken: string;
+
+  /**
+   * Name of the sub verifier in case of aggregate verifier setup. This field should only be provided in case of an aggregate verifier.
+   */
+  subVerifier?: string;
+
+  /**
+   * Extra verifier params in case of a WebAuthn verifier type.
+   */
+  extraVerifierParams?: WebAuthnExtraParams;
+
+  /**
+   * Any additional parameter (key value pair) you'd like to pass to the login function.
+   */
+  additionalParams?: ExtraParams;
+}
+
 // TODO extend interface and type documentation wherever necessary. @Yash
 export interface ICoreKit {
-  /** The tKey instance, if initialized. */
+  /** The tKey SDK instance
+   *  TKey is the core module on which this wrapper SDK sits for easy integration.
+   **/
```

Comment:
This needs to be formatted according to standard.
```suggestion
  /**
   * The tKey instance, if initialized.
   * TKey is the core module on which this wrapper SDK sits for easy integration.
   **/
```